src/components/layout/Navbar.jsx

```diff
@@ -0,0 +1,44 @@
+import { React, useState, useEffect } from 'react';
+
+
+function Navbar() {
+    const [isMobileMenuOpen, setIsMobileMenuOpen] = useState(false);
+
+
+    const toggleMobileMenu = () => {
+        setIsMobileMenuOpen(!isMobileMenuOpen);
+    };
+
+
+
+    return (
+
+        <header
+            className={`header  ${isMobileMenuOpen ? 'nav-open' : ''}`}
+            style={{
+                zIndex: '999',
+                position: 'sticky',
+                top: '0',
+                backdropFilter: 'blur(10px)',
+                backgroundColor: 'rgba(255, 255, 255, 0.7)',
+            }}
+        >
+            <div className="logo">  <a href="#">
+                <img className="logo" alt="Be natural logo" src="./logo.svg" style={{ height: '4.5rem', width: '3rem', position: 'absolute', top: '50%', transform: 'translateY(-50%)' }} />
+            </a></div>
+            <div className="heading-primary" style={{ position: 'absolute', left: '50%', transform: 'translateX(-50%)' }}>JobTracker</div>
+            <nav className={`main-nav ${isMobileMenuOpen ? 'open' : ''}`} style={{ zIndex: '300' }}>
+                <ul className="main-nav-list" style={{ fontFamily: "Rubik" }}>
+                    <li><a className="main-nav-link" href="/" style={{ fontFamily: "Rubik" }}>About</a></li>
+                    <li><a className="main-nav-link" href="https://github.com/madina0801/JobTrackr" >GitHub</a></li>
```

Comment:
Sorry, forgot to mention the GitHub page should be opened in new browser tab (target=”_blank")